.github/workflows/api-release.yml

```
@@ -0,0 +1,64 @@
+          SIGN_KEY_PASS: ${{ secrets.gpg_passphrase }}
+          CENTRAL_USERNAME: ${{ secrets.sonatype_username }}
+          CENTRAL_PASSWORD: ${{ secrets.sonatype_password }}
+
+  create-github-release:
+    if: ${{ !failure() && !cancelled() }}
+    needs: "release-maven"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub Release
+        id: create_release
+        uses: softprops/action-gh-release@da05d552573ad5aba039eaac05058a918a7bf631 # v2.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)